app/src/main/res/values/styles.xml

```diff
@@ -25,4 +25,12 @@
     <style name="SplashTheme" parent="Theme.AppCompat.NoActionBar">
         <item name="android:windowBackground">@drawable/background_splash</item>
     </style>
+
+    <style name="AccentButton" parent="ThemeOverlay.AppCompat.Dark">
```

Comment:
Why do we need this?

Reply:
Because by default, the enabled state changes color of button to accentColor, but we need it to be primaryColor. So I use this to change colors for enabled and disabled states of button.

Reply:
Use backgroundTint

Reply:
changing backgroundTint from code requires passing ColorStateList, in which I will have to define color for each state. i.e. pressed, unpressed, enabled, disabled, etc. This  seemed better. If you want I can use ColorStateList.

Reply:
backgroundTint="?attr/colorPrimary"

Reply:
Oh, in this style, ok

Reply:
Tried, if I set backgroundTint, the color doesn't change to grey when button is disabled